### DISCUSSION TITLE
Content library, HA, DRS and affinity rules support

### DISCUSSION THREAD

**Discussion by cathode911** [saltstack/salt#67283] [Feature Requests]
### Description of Issue
Hello!
We're trying to migrate to salt-cloud from terraform and found some  important features missing in salt-cloud such as: 

- VM deployment from content library (also missing in terraform, although request exists from 2018)
- creation / deletion of affinity or anti-affinity DRS rules (top priority for us)
- VM High-Availability restart priority change
- creation / deletion of Resource Pools

It would be great to have those features in salt-cloud and hopefully will help lost of users to achieve their infrastructure goals

### Setup
Any salt-cloud setup

### Steps to Reproduce Issue
Unreproducible due to non existing features

### Versions Report
Any

**Comment by Arderos** [comment]
+1. These would be of extreme benefit for us as well

**Comment by cathode911** [comment]
it has been over 3 years now.
Any chance of considering at least creation / deletion of affinity or anti-affinity DRS rules ?
Setting tags for VMs has also huge benefit, tags are widely used for variety of purposes

**Comment by das-ich** [comment]
we need these features too
